src/mode/meta.js

```diff
@@ -1,5 +1,6 @@
 import CodeMirror from "codemirror/lib/codemirror"
+import * as ABAP from "./abap/abap.js";
```

Comment:
hmm, yeah, wanted to be able to test the abap mode

perhaps we can merge this PR, and then look into the rollup setup? I'm not really sure how to set it up so it all works within the github setup